app/javascript/test/coverage/clover.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We shouldn't include coverage in the code - can you gitignore and remove all these?

Reply:
👍